### PR TITLE
Chore/Add readme content

### PR DESCRIPTION
**Pull request by friedger**
This PR adds short readme content for all projects.

For the first demos, I fixed some issues to make them work. At the end, I just added the readme files so we can decide which demos are important to us.

### CODE REVIEW

**Comment by friedger** [comment]
waiting for #8

**Comment by friedger** [comment]
merging master

File: PushNotifications/c2dm_urban_airship/README.md [hunk 1]
```diff
@@ -1,2 +1,68 @@
...
+
+##Step 4:
+
+In order to get the C2DM authorisation token you need to execute a curl command
+
+curl https://www.google.com/accounts/ClientLogin -d Email=your_user -d "Passwd=your_password" -d accountType=GOOGLE -d source=Google-cURL-Example -d service=ac2dm
+
+Enter your password and email in the required fields. The response will resemble this
+
+SID=DQAAAPMAAACMY6LUPhXt_co8ESnZa9_bX17MJJZj2AGlDr4gcNIXxq7OJBMPFiT8ry756bE_0rmx
```
Comment (xrigau) [inline PushNotifications/c2dm_urban_airship/README.md]: I guess this is not a real token?

Comment (friedger) [inline PushNotifications/c2dm_urban_airship/README.md]: That was there before
Am 30.06.2015 12:59 schrieb "Xavi Rigau" <notifications@github.com>:

**Comment by friedger** [comment]
thanks

**Pull request merged**